src/components/frame/v5/pages/VerifiedPage/partials/MemberAvatar/MemberAvatar.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from 'react';
+import { useContributorBreakdown } from '~hooks';
+
+import { splitWalletAddress } from '~utils/splitWalletAddress';
+import UserAvatarPopover from '~v5/shared/UserAvatarPopover';
+import { MemberAvatarProps } from './types';
+
+const MemberAvatar: FC<MemberAvatarProps> = ({ member }) => {
+  const { user } = member || {};
+  const { walletAddress = '', profile } = user || {};
+  const { bio, displayName } = profile || {};
```

Comment:
bio = ''

Reply:
It wasn't working like this, I left it how it was